src/Microsoft.AspNet.OData.Shared/IPerRouteContainer.cs

```diff
@@ -11,6 +11,20 @@ namespace Microsoft.AspNet.OData
     /// </summary>
     public interface IPerRouteContainer
```

Comment:
>public [](start = 4, length = 6)

I assume there are no custom implementations that would break by our adding members to this interface?

Reply:
Same as the comment in "IODataFeature".

---
In reply to: [384697141](https://github.com/OData/WebApi/pull/2035#discussion_r384697141) [](ancestors = 384697141)